_posts/2017-01-31-kap.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Project of the Week: Kap
+author: zeke
+---
+
+The Electron community is growing quickly, and people are creating powerful
+new apps and tools at an astounding rate. To celebrate this creative momentum
+and keep the community informed of some of these new projects, we've decided to
+start a weekly blog series featuring noteworthy Electron-related projects.
+
+---
+
+This post is the first in the series, and features [Kap](https://getkap.co/),
+an open-source screen recording app built by [Wulkano](https://wulkano.com/),
+a geodistributed team of freelance designers and developers.
+
+[![Kap Screencast](https://cloud.githubusercontent.com/assets/2289/22439463/8f1e509e-e6e4-11e6-9c32-3a9db63fc9a1.gif)](https://getkap.co/)
+
+## What is Kap and why did you create it?
+
+[Kap is an open-source screen recorder](https://getkap.co) built primarily for designers and developers to easily capture their work. People use it to share animated prototypes, document bugs, create silly GIFs and everything in-between.  
+
+We've seen people of all ages and backgrounds use it in educational settings, screencasts, tutorials... the list goes on. Even to create production assets! We're completely blown away by how well received our little side project has been.
+
+Why did we build it? That's a very good question, it's not like there's a lack of screen recorders out there! We felt the alternatives were either too complex, too expensive or too limited. Nothing felt *just right* for our everyday needs. We also think it's great when the tools we use to do our work are open-source, that way everyone can help shape them. [Building Kap ended up being just as much about what we didn't do](https://medium.com/wulkano-friends/from-idea-to-product-and-beyond-a12850403c38). It's all in the details, an accumulation of small improvements that became the outline of a tool we wanted to use.
```

Comment:
Ohh.. hehe.. I see. They repeated the 2nd question in their answer, notice now the "we".

> Maybe we should split it up into two questions?

Yeah, might be more straightforward.